packages/screens/FeedPostView/components/FeedPostArticleMarkdownView.tsx

```diff
@@ -0,0 +1,397 @@
+
+const contentPaddingHorizontal = layout.spacing_x2;
+
+export const FeedPostArticleMarkdownView: FC<{
+  post: Post;
+  refetchPost: () => Promise<any>;
+  isLoadingPost?: boolean;
+}> = ({ post, refetchPost, isLoadingPost }) => {
+  const navigation = useAppNavigation();
+  const { width: windowWidth } = useWindowDimensions();
+  const { width } = useMaxResolution();
+  const isMobile = useIsMobile();
+  const [parentOffsetValue, setParentOffsetValue] = useState(0);
+
+  const authorId = post?.authorId;
+  const authorNSInfo = useNSUserInfo(authorId);
+  const [, authorAddress] = parseUserId(post?.authorId);
+  const username = authorNSInfo?.metadata?.tokenId || authorAddress;
+
+  const [localPost, setLocalPost] = useState(post);
+  const feedInputRef = useRef<NewsFeedInputHandle>(null);
+  const [replyTo, setReplyTo] = useState<ReplyToType>();
+  const aref = useAnimatedRef<Animated.ScrollView>();
+  const [flatListContentOffsetY, setFlatListContentOffsetY] = useState(0);
+  const [articleOffsetY, setArticleOffsetY] = useState(0);
+  const [articleWidth, setArticleWidth] = useState(0);
+  const [renderHtmlWidth, setRenderHtmlWidth] = useState(0);
+  const isGoingUp = useSharedValue(false);
+  const isLoadingSharedValue = useSharedValue(true);
+  const [isCreateModalVisible, setCreateModalVisible] = useState(false);
+  const {
+    data: comments,
+    refetch: refetchComments,
+    hasNextPage,
+    fetchNextPage,
+    isLoading: isLoadingComments,
+  } = useFetchComments({
+    parentId: post.id,
+    totalCount: post.subPostLength,
+    enabled: true,
+  });
+  const isNextPageAvailable = useSharedValue(hasNextPage);
+
+  const articleMetadata = zodTryParseJSON(
+    ZodSocialFeedArticleMarkdownMetadata,
+    post.metadata,
+  );
+  const message = articleMetadata?.message;
+  const html = message ? md.render(message) : null;
+  const title = articleMetadata?.title;
+  const location = articleMetadata?.location;
+
+  const headerLabel = useMemo(() => {
+    const authorDisplayName =
+      authorNSInfo?.metadata?.tokenId ||
+      tinyAddress(authorAddress) ||
+      DEFAULT_USERNAME;
+    return `Article by ${authorDisplayName}`;
+  }, [authorNSInfo?.metadata?.tokenId, authorAddress]);
+
+  const onPressReply: OnPressReplyType = (data) => {
+    feedInputRef.current?.resetForm();
+    setReplyTo(data);
+    feedInputRef.current?.setValue(`@${username} `);
+    feedInputRef.current?.focusInput();
+  };
+
+  const handleSubmitInProgress = () => {
+    if (replyTo?.parentId && replyTo.yOffsetValue)
+      aref.current?.scrollTo(replyTo.yOffsetValue);
+    else aref.current?.scrollTo(0);
+  };
+
+  const scrollHandler = useAnimatedScrollHandler(
+    {
+      onScroll: (event) => {
+        let offsetPadding = 40;
+        offsetPadding += event.layoutMeasurement.height;
+        if (
+          event.contentOffset.y >= event.contentSize.height - offsetPadding &&
+          isNextPageAvailable.value
+        ) {
+          fetchNextPage();
```

Comment:
IMO, this code will be triggered every time onScroll event is fired, we should add isFetching flag to block that

Reply:
`FeedPostArticleMarkdownView` is a copy of `FeedPostArticleView`.
We need to refacto the posts UI code. I don't want this PR being messy, the purpose is just to add a Markdown editor. But it implies to add a new PostCategory, so a new Zod type, so a new FeedPostBlablaView...

Reply:
I completed the description

>  Please take attention to that: At the final, we'll have only one Article category, type, view, etc. These stuff will be merged with the actual "Article" stuff, and the mention "Markdown" will be removed in the future